mentorship/gsoc/2022/projects/new-docs-site-engine/README.md

```diff
@@ -1,49 +1,352 @@
+
+**Project Idea:** [https://github.com/keptn/community/blob/main/mentorship/gsoc/2022/project-ideas.md#new-documentation-site-engine](https://github.com/keptn/community/blob/main/mentorship/gsoc/2022/project-ideas.md#new-documentation-site-engine)
+
+## Work Summary
+
+I had divided the Development Phase into two phases i.e. `Phase 1` and `Phase 2`.
+
+> `Phase 1` was to deal mostly with implementing core features and functionality needed in the documentation engine.
+
+> `Phase 2` was to deal mostly with implementing miscellaneous features and functionality of the documentation engine like Continuous Integration, Continuous Deployment, etc.
+
+#### Phase 1
+
+> **Week 1 - Week 6 (June 13 - July 25, 2022)**
+
+---
+
+- [x] Initialization and deployment of the documentation engine based on Docusaurus from scratch.
+
+  - [x] Initialised the new Docusaurus site using `npx create-docusaurus@latest new-keptn-docs-engine classic` command and push the changes to [keptn-sandbox/new-keptn-docs-engine](https://github.com/keptn-sandbox/new-keptn-docs-engine).
+
+    > Above task is done and I have pushed the changes directly to the repository. Commit can be found here [https://github.com/keptn-sandbox/new-keptn-docs-engine/commit/e2f2202b01894806e1d9e924d97abb2b98b2d8d1](https://github.com/keptn-sandbox/new-keptn-docs-engine/commit/e2f2202b01894806e1d9e924d97abb2b98b2d8d1).
+
+  - [x] Adding as basic documentation on how to use Docusaurus and what configuration of Docusaurus we are following like Project structure, Templates, etc.
+
+    > I used the `npx create-docusaurus@latest new-keptn-docs-engine classic` command to create the new Docusaurus site so in this case, I used the `classic` template. Docusaurus recommend the `classic` template so that we can get started quickly, and it contains features found in Docusaurus 1. The `classic` template contains `@docusaurus/preset-classic` which includes standard documentation, a blog, custom pages, and a CSS framework (with dark mode support). We can get up and running extremely quickly with the classic template and customize things later on when you have gained more familiarity with Docusaurus. We can also use the template's TypeScript variant by passing the `--typescript` flag. See [TypeScript support](https://docusaurus.io/docs/typescript-support) for more information.
+
+  - [x] Deploying the new Docusaurus site engine to Netlify.
+
+    > [https://github.com/keptn-sandbox/new-keptn-docs-engine](https://github.com/keptn-sandbox/new-keptn-docs-engine) has been deployed to Netlify at [https://keptn-experimental-docs-site.netlify.app/](https://keptn-experimental-docs-site.netlify.app/).
+
+  - [x] Adding `CONTRIBUTING.md` file with content related to how to add contents in Docusaurus and how to use it concerning Keptn. Also, this work will go on since we will be adding more content in the future according to documentation needs.
```

Comment:
The "Built with" list in CONTRIBUTING.md should be updated  to include the other things you have implemented - Lighthouse CI, Vale linter (currently using Google style guide), etc.  No need to fill in details about how to use them and such -- just a chance to highlight all the stuff you got working in this project.